graspy/simulations/simulations_corr.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+from graspy.simulations import sample_edges
+
+
+def sample_edges_corr(P, R, directed, loops):
+    """
+    Generate a pair of correlated graphs with Bernoulli distribution.
+    Both G1 and G2 are binary matrices. 
+
+    Parameters
+    ----------
+    P: np.ndarray, shape (n_vertices, n_vertices)
+        Matrix of probabilities (between 0 and 1) for a random graph
+    R: np.ndarray, shape (n_vertices, n_vertices)
```

Comment:
you need to do input checks, mainly just for R. What possible values of correlation are allowed? What if someone inputs a string instead of a matrix?